src/components/section/feature/FeatureDetailBody.jsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { getDetails } from "@/components/common/fetchData";
+import Image from "next/image";
+
+const FeatureDetailBody = ({ data }) => {
+
+  return (
+    <section className="c-feature">
+      <h2 className="c-heading--lv2">{data.subject}</h2>
+      <div className="c-feature__intro">
+        {data.ext_2}
+      </div>
+      <ul className="c-feature__list">
+        {data.ext_3.map(async (item, index) => {
```

Comment:
@VortexExpansion 
Pls avoid using async fn as an inline one due to causing heavy performance issue.
Use Promise.all() and `useMemo()/useCallback()` instead.

Reply:
@yabe-diverta san

I used Promise.all and useCallBack and removed it from being inline 
https://github.com/diverta/front_next_media/pull/5/commits/2308cdeaa1ac6cb3a865bda7e6716af071ebe648

But I noticed that the performance is now worse and it takes more time to fetch and load. Can you please review this once 
https://diverta.gyazo.com/b40e8ea6ebd41613ad5c3d44ebeceb60

I felt that the load times of the code before was much faster https://diverta.gyazo.com/6ab6ba2db39b9688b16d0072792f9e0c

Reply:
Far more checking, I think this client component should not fetch data, changed to fetch data at server and pass the data as props.
pls check [this update](https://github.com/diverta/front_next_media/pull/5/commits/2238cea1d4d0deda329925eca6c71a344da1b4c6) 🙇